src/stores/Object.js

```diff
@@ -221,6 +221,35 @@ class ObjectStore {
     return this.writeTokens[objectId];
   });
 
+  @action.bound
+  CopyObjectPermission = flow(function * ({libraryId, objectId}) {
+    // Determine if owner
+    const owner = Fabric.utils.FormatAddress(
+      yield Fabric.client.CallContractMethod({
+        contractAddress: Fabric.client.utils.HashToAddress(objectId),
+        methodName: "owner"
+      })
+    );
+    const isOwner = EqualAddress(owner, yield Fabric.CurrentAccountAddress());
+
+    if(isOwner) return true;
```

Comment:
Personal preference - I don't like bare statements with if clauses. please wrap this in brackets.